### PR TITLE
Delete processes and services

### PR DESCRIPTION
**Pull request by IronGauntlets**
## Changes:
- Delete processes and services to make it easier to run and maintain juno. Services were implemented as singletons which lead to side effects when they are used. The process struct provided a concurrency wrapper around the HTTP server and service calls which lead to concurrent bugs, such as panic on a negative wait group call. 

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [x] No

### CODE REVIEW

File: cmd/juno/cli/root.go [hunk 1]
```diff
@@ -8,25 +8,41 @@ import (
...
+
+	"github.com/NethermindEth/juno/internal/db/state"
+
+	"github.com/NethermindEth/juno/internal/db/abi"
+
+	"github.com/NethermindEth/juno/internal/db/contracthash"
+	"github.com/torquem-ch/mdbx-go/mdbx"
+
+	"github.com/NethermindEth/juno/pkg/rest"
+	"github.com/NethermindEth/juno/pkg/starknet"
```
Comment (tshakalekholoane) [inline cmd/juno/cli/root.go:27]: Why is there space between imports?

File: pkg/starknet/state.go [hunk 2]
```diff
@@ -10,11 +10,18 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/NethermindEth/juno/internal/db/block"
+
+	"github.com/NethermindEth/juno/internal/db/transaction"
+
+	"github.com/NethermindEth/juno/internal/db/state"
+
```
Comment (tshakalekholoane) [inline pkg/starknet/state.go:18]: See comment above.

> Why is there space between imports?

**Review by tshakalekholoane — CHANGES_REQUESTED** [review]
See comments. Also, coverage issues can be fixed by adding a `// notest` comment in the required block.

File: cmd/juno/cli/root.go (hunk 1 above)
Comment (IronGauntlets) [reply cmd/juno/cli/root.go]: Yeah, its to do with the file watcher I used for goland. Goland support for gofumpt adds these spaces when it rearranges the imports.

Comment (IronGauntlets) [reply cmd/juno/cli/root.go]: Please view the latest commit and please resolve the conversation

File: pkg/starknet/state.go (hunk 2 above)
Comment (IronGauntlets) [reply pkg/starknet/state.go:18]: same as above

**Review by tshakalekholoane — DISMISSED** [review]
Brilliant work!

**Comment by tshakalekholoane** [comment]
Sorry about that 😵 the `// notest` has to go right under the closure that Codecov is flagging i.e. just right above where you see banner in the details section.

**Review by tshakalekholoane — APPROVED** [review]

**Review by adriantpaez — APPROVED** [review]

**Pull request merged**